### PR TITLE
 [type: feature] add http file to examples[grpc,http,spring cloud]，fix pr [#2059]

### PR DESCRIPTION
**Pull request by li-keguo**
add http file to examples[grpc,http,spring cloud] [#2044]

other examples project learning, subsequent submission

work list:

 add http file to 
-  add http file to  examples[grpc,http,spring cloud]
- remove HttpServerConfig#environment [shengyu-examples-http], it never used
- add ShenyuWebHandler#ShenyuWebHandler()[shengyu-web] java docs, it build warning

### CODE REVIEW

File: shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public UserDTO findByUserId(@RequestParam("userId") final String userId) {
     public UserDTO getPathVariable(@PathVariable("id") final String id, @RequestParam("name") final String name) {
         UserDTO userDTO = new UserDTO();
         userDTO.setUserId(id);
-        userDTO.setUserName("hello world");
+        userDTO.setUserName("hello world" + name);
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java:80]: donot modify this

Comment (li-keguo) [reply shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java:80]: ok

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java [hunk 2]
```diff
@@ -113,7 +112,7 @@ public void testGetAndSet() throws Exception {
                     try {
                         // get the get and set methods of the field by PropertyDescriptor
                         // (String) f.getName() for java11 (can not find com.sun.beans.introspect.PropertyInfo class)
-                        PropertyDescriptor property = new PropertyDescriptor((String) f.getName(), clazz);
+                        PropertyDescriptor property = new PropertyDescriptor(f.getName(), clazz);
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java:115]: pls do not modfiy this

**Comment by yu199195** [comment]
@li-keguo  can you fix this pr？  

I suggest you only add grpc-test-api.http / springcloud-test-api.http/ http-test-api.http

**Comment by li-keguo** [comment]
ok

**Comment by li-keguo** [comment]
I encountered a problem in adding `tars` and `sofa`, which is being solved

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java (hunk 2 above)
Comment (li-keguo) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java]: ok

**Review by yu199195 — APPROVED** [review]

**Pull request merged**